src/forms/userForm.tsx

```diff
@@ -181,11 +176,11 @@ class UserForm extends React.Component<UserFormProps, UserFormState> {
                       show_asset_options: false,
                     });
                   }}
-                ></Checkbox>
+                />
```

Comment:
Nice.

Reply:
man whoever wrote `<Checkbox ></Checkbox>` for ev3 must feel like a clown rn

Reply:
HAHA I didn't even know what this but pycharm was really yelling at me for it